### ISSUE TITLE
e not behaving as expected

### ISSUE THREAD

**Issue by jtmkrueger** [seanyeh/vibreoffice#3]
When I press `e` in vim, it behaves the same as `w` but puts me at the end of successive words. `e` is only jumping me to the end of the word I'm on, then it doesn't work anymore.

**Comment by seanyeh** [comment]
Yes, word and sentence movement is sort of broken at the moment :( I'm just using the LibreOffice word/sentence movement API, and its behavior is different from vi's. Unfortunately, I've been extremely busy these days so I haven't been able to spend much time working on it. I'll try my best to get to it someday, but if you can't wait, feel free to make a pull request!
